public/app/features/dashboard/components/DashNav/DashNavButton.tsx

```diff
@@ -8,7 +8,7 @@ import { GrafanaTheme } from '@grafana/data';
 interface Props {
   icon?: IconName;
   tooltip: string;
-  onClick?: () => void;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
Please add MouseEvent to react import instead.
```suggestion
  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
```

Reply:
@hugohaggmark Oh, you are right! The thing is that we have forked Grafana and heavily customized it for one of our internal projects, and I've just thought why not to try some tiny PR... By the way, your change request is done.

Reply:
Ok, thank you for clarifying that and fixing the last thing.